GitCommands/CommitTemplateItem.cs

```diff
@@ -90,8 +90,10 @@ private static string SerializeCommitTemplates(CommitTemplateItem[]? items)
 
                     byte[] memoryData = Convert.FromBase64String(serializedString.Substring(p + 1));
                     using var rs = new MemoryStream(memoryData, 0, length);
+#pragma warning disable SYSLIB0011 // Type or member is obsolete
```

Comment:
📝 We should consider filing issues to migrate these to another serializer

Reply:
Yes, will do once this is merged.
I had a brief look at the code, and looks like this serialization may have been used originally before it was updated to json-based.

Reply:
https://github.com/gitextensions/gitextensions/issues/9150